### DISCUSSION TITLE
sentinel-1-rtc requires authentication

### DISCUSSION THREAD

**Discussion by gtgrp-user** [microsoft/PlanetaryComputer#167] [General]
Do you know if there is any sample python scripts to load sentinel-1-rtc collection in a xarray using authentication?

`ds = odc.stac.load(item_collection,
               measurements=['vv', 'vh'],
               chunks={"x": 2048, "y": 2048},
               bbox=geom.bounds,
               stac_cfg=cfg,
               patch_url=pc.sign
)`

It tried above which worked for sentinel-2, however, throws the following error for sentinel-1,

`HTTPError: 403 Client Error: Forbidden for url: https://planetarycomputer.microsoft.com/api/sas/v1/token/sentinel1euwestrtc/sentinel1-grd-rtc`

**Comment by TomAugspurger** [comment]
Some datasets, like sentinel-1-rtc, require you to provide your Planetary Computer API key.  From https://planetarycomputer.microsoft.com/dataset/sentinel-1-rtc

> A Planetary Computer account is required to retrieve SAS tokens to read the RTC data. See the [documentation](http://planetarycomputer.microsoft.com/docs/concepts/sas/#when-an-account-is-needed) for more information.

https://planetarycomputer.microsoft.com/docs/concepts/sas/#when-an-account-is-needed goes into detail, but if you're using the `planetary-computer` the easiest way to set the `PC_SDK_SUBSCRIPTION_KEY ` or run `planetarycomputer configure`.

**Reply by gtgrp-user** [reply]
Thanks a lot.
